cli/templateedit.go

```diff
@@ -57,6 +59,7 @@ func templateEdit() *cobra.Command {
 	}
 
 	cmd.Flags().StringVarP(&name, "name", "", "", "Edit the template name")
```

Comment:
What do you think about changing `name` to `slug` instead? We could make this naming consistent as well across all objects. This can happen in another PR, just thinking about the API moving forward.

Reply:
That's an interesting idea, let me think loud...

We could define a policy that every database object has a mandatory unique ID, unique slug, and optional display name. Potentially slugs can be used as source data for a global search feature and we won't need to index descriptions, like in OSX cmd+space.

> This can happen in another PR, just thinking about the API moving forward.

Yup, I can raise an issue and we can continue the discussion there. Otherwise, it will be scattered across different comments :)